packit_service/worker/events/gitlab.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         target_repo_branch: str,
         project_url: str,
         commit_sha: str,
+        oldrev: str,
```

Comment:
I think the type of this should be `Optional[str]`, b/c if there is no code change, then the `nested_get` bellow is going to return `None`. See [GitLab API docs](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#merge-request-events): "The field `object_attributes.oldrev` is only available when there are actual code changes..."

Reply:
```suggestion
        oldrev: Optional[str],
```